codespeed/templates/codespeed/timeline.html

```diff
@@ -3,6 +3,7 @@
 {% block title %}{{ block.super }}: Timeline{% endblock %}
 
 {% block extra_head %}
+{{ block.super }}
```

Comment:
I wonder, did this fix something for you? Since `timeline.html` extends `base.html` and the extra_head block is empty in the base, this won't really do anything as far as I can see.